tests/robustness/traffic/kubernetes.go

```diff
@@ -129,7 +129,7 @@ func (t kubernetesTraffic) Read(ctx context.Context, kc kubernetes.Interface, s
 		}
 		hasMore = resp.Count > int64(len(resp.Kvs))
 		if hasMore {
-			cont = string(kvs[len(kvs)-1].Key)
+			cont = string(kvs[len(kvs)-1].Key) + "\x00"
```

Comment:
https://github.com/etcd-io/etcd/blob/release-3.5/client/v3/kubernetes/interface.go#L39-L40

> The Continue value should be the last key returned in a previous paginated ListResponse

Reply:
I talked to Marek. The fix is correct, but the documentation is wrong. I opened https://github.com/etcd-io/etcd/pull/19281 to fix the discrepancy